server/app/modules/hunts/router.py

```diff
@@ -26,4 +27,15 @@ async def load_past_hunts():
   return HuntResponseModel(
     message="fetched hunts",
     content=result
+  )
+
```

Comment:
Don't know why this is called "join hunt"
The client can have a page called join hunt but the server should have a name that makes sense on its own